tutorxqueue/templates/xqueue/build/xqueue/Dockerfile

```diff
@@ -1,5 +1,5 @@
 FROM docker.io/ubuntu:20.04
-MAINTAINER Overhang.io <contact@overhang.io>
+LABEL maintainer="Overhang.io <contact@overhang.io>"
```

Comment:
You can simply get rid of this field, as discussed here: https://github.com/overhangio/tutor-forum/issues/12#issuecomment-1314976659

Could you also please upgrade the uwsgi version below? https://pypi.org/project/uWSGI/

Reply:
@regisb done